src/tribler/gui/single_application.py

```diff
@@ -89,11 +89,12 @@ def _on_new_connection(self):
         self._incoming_stream.setCodec('UTF-8')
         connect(self._incoming_connection.readyRead, self._on_ready_read)
         if self.tribler_window:
-            self.tribler_window.restore_from_minimised()
+            self.tribler_window.raise_window()
```

Comment:
I assume this could be platform dependent. Does it work on three major platforms (Win, macOS, Linux)?